docs/application-development-guide.md

```diff
@@ -2513,7 +2513,7 @@ run().catch((err) => {
 
 #### Set Workflow Id
 
-Also it is not required, we recommend providing your own [Workflow Id](/workflows/#workflow-id) that maps to a business process or business entity identifier, such as an order identifier or customer identifier.
+While it is not required, we recommend providing your own [Workflow Id](/workflows/#workflow-id) that maps to a business process or business entity identifier, such as an order identifier or customer identifier.
```

Comment:
```suggestion
Although it's not required, we recommend providing your own [Workflow Id](/workflows/#workflow-id) that maps to a business process or business entity identifier, such as an order identifier or customer identifier.
```
"While" -> "Although" per "while" in https://developers.google.com/style/word-list#letter-w.